docs-src/references/dynamic-configuration.md

```diff
@@ -77,33 +77,34 @@ Not defining constraints on a dynamic configuration key sets the values globally
   ```yaml
   frontend.rps: # Rate limit on the number of requests sent to the Frontend host.
     - constraints: {} # Sets default value that applies to all Namespaces
-      value: 2000 # The defaul value for this key on the history host is 3000.
+      value: 2000 # The default value for this key on the Frontend host is 3000.
   - constraints: {namespace: "namespace1"} # Sets value for "namespace1" Namespace
-      value: "4000"
+      value: 4000
     - constraints: {namespace: "namespace2"}
-      value: "1000"
+      value: 1000
   ```
 
 - For keys that can be customized at a Task Queue level, you can specify Task Queue name and Task type in addition to Namespace.
   To set values at a Task Queue level, use `taskqueueName` (String) with `taskType` (optional; supported values: `Workflow` and `Activity`).
 
-  For example to override the default maximum queries per second made to the Persistence database from the Frontend Service, add the following to your dynamic configuration file.
+  For example if you have Workers creating a large number of Workflow and Activity tasks per second, you can add more partitions to your Task Queues (default is 4) to accomodate the high number of polls.
+  To do this, add the following to your dynamic configuration file.
 
   ```yaml
-  frontend.persistenceMaxQPS:
+  matching.numTaskqueueReadPartitions:
```

Comment:
sorry, maybe this was a bad example after all... if you set this you also have to set `matching.numTaskqueueWritePartitions` to the same value. so the example will be twice as long :(

Reply:
I don't mind a longer example tbh (and it isn't terribly long). Added `matching.numTaskqueueWritePartitions` to the example with comments and a preceding note: "Note that if changing the number of partitions, you must set the same count for both read and write operations on Task Queues."